recipes/godot-cpp/all/conanfile.py

```diff
@@ -12,20 +20,81 @@ class GodotCppConan(ConanFile):
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/godotengine/godot-cpp"
     topics = ("game-engine", "game-development", "c++")
+
+    package_type = "static-library"
     settings = "os", "arch", "compiler", "build_type"
-    build_requires = ["scons/3.1.2"]
+    options = {
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "fPIC": True,
+    }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires(f"godot_headers/{self.version}", transitive_headers=True)
+
+    def package_id(self):
+        if self.info.settings.build_type != "Debug":
+            self.info.settings.build_type = "Release"
```

Comment:
Why is this needed? https://github.com/godotengine/godot-cpp/blob/master/CMakeLists.txt#L2 seems to imply this is an option. Does it actually always produce the same resulting binaries?

Reply:
`RelWithDebInfo` and other less common build types get built as `Release` by the project, so they are all equivalent. `Debug` is the only exception.

Reply:
Oh I missed that its looking for soemthing _other than_ debug  missed the negation. Thanks for clarifying